tools/devctr/pyproject.toml

```diff
@@ -5,31 +5,30 @@ description = "Firecracker python dependencies"
 authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
-aws-embedded-metrics = "*"
-black = "*"
-boto3 = "*"
-botocore = "*"
-dataclasses = "*"
+aws-embedded-metrics = "^3.1.0"
+black = "^23.3.0"
+boto3 = "^1.26.140"
+botocore = "^1.29.140"
 # We need to fix the gitlint version since version 0.17.0 has a problem:
 # https://pythonissues.com/issues/1410824
-gitlint = "0.16.0"
-isort = "*"
-nsenter = "*"
-openapi-spec-validator = "*"
-pandas = "*"
-psutil = "5.*"
-pylint = "*"
-pytest = "*"
-pytest-json-report = "*"
-pytest-timeout = "*"
-pytest-xdist = "*"
+gitlint = "^0.19.1"
```

Comment:
This disagrees with the comment above. If the problem is now fixed, could you remove the comment?

Reply:
Removed the comment.